minos/common/database/database.py

```diff
@@ -33,13 +32,9 @@ def __init__(self, host: str, port: int, database: str, user: str, password: str
         self.database = database
         self.user = user
         self.password = password
-        self._pool = None
-
-    async def _destroy(self) -> NoReturn:
-        if self._pool is not None:
-            self._pool.close()
-            await self._pool.wait_closed()
-            self._pool = None
+        self._pool = AiopgPool(
```

Comment:
Why not to instead of setting the `host`, `port`, `database`, etc. attributes here, we propagate it to the `AiopgPool` class and store them on it?

Reply:
Done